src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/AdjudicationTransformer.kt

```diff
@@ -3,22 +3,13 @@ package uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer
 import org.springframework.stereotype.Component
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.Adjudication
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.model.prisonsapi.AdjudicationsPage
-import java.time.LocalDate
 import java.time.ZoneOffset
 
 @Component
-@SuppressWarnings("MagicNumber")
 class AdjudicationTransformer {
-  fun transformToApi(adjudicationsPage: AdjudicationsPage, getLast12MonthsOnly: Boolean) =
+  fun transformToApi(adjudicationsPage: AdjudicationsPage) =
     adjudicationsPage.results.flatMap { result ->
       result.adjudicationCharges
-        .filter {
-          if (getLast12MonthsOnly) {
```

Comment:
you appear to have removed the functionality the feature flag was enabling? I think 'getLast12MonthsOnly' should be true if the caller is for approved premises?